player/player.js

```diff
@@ -1,6 +1,9 @@
+/* global $ Papa */
+/* eslint no-console:0 */
+
 var stepped = 0, chunks = 0, rows = 0;
 var start, end;
-var parser;
+var parser; // eslint-disable-line no-unused-vars
```

Comment:
Not sure if this is used as part of `player.html` so I kept it in.

Reply:
I will prefer to keep player as is and ignore it.